### PR TITLE
Add markdown linting GitHub Action

### PR DESCRIPTION
**Pull request by CBielstein**
This adds markdown linting GitHub Action to simplify review of markdown changes in new PRs. Can be used as a reference for #9 

**How verified**:

* Pushed a commit with no style violations and it passed
* Pushed a commit with a linting style violation and it failed

### CODE REVIEW

File: .github/workflows/markdown.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Lint
+
+on:
+  push:
+    branches:
+      - main
+
+  pull_request:
+    branches:
+      - main
```
Comment (lugolven) [inline .github/workflows/markdown.yml:10]: let's not waste you precious github actions minutes:
````yaml
   paths:
      - '**/*.md'
````

https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths

Comment (CBielstein) [reply .github/workflows/markdown.yml:10]: Well, minutes are free for public repositories... :)

Separately, there's something simple for every check running on every push or PR. The code must always be clean for a PR to merge. If a check if failing, such a system keeps reminding the devs of the problem. On the other hand, for something like a simple markdown linting issue, perhaps it's not worth stopping a PR at all....

I'll consider this. I'm interested if you have thoughts on running checks every time or ad hoc.

File: .github/workflows/markdown.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: Lint
```
Comment (CBielstein) [inline .github/workflows/markdown.yml:1]: I think this should be named markdown instead of lint.

File: .github/workflows/markdown.yml (hunk 1 above)
Comment (CBielstein) [reply .github/workflows/markdown.yml:10]: After further thought, I decided to take your advice for PRs. No need to run this on PRs that don't touch the markdown files. However, I'll leave it running on every push to main.

File: README.md [hunk 3]
```diff
@@ -4,23 +4,32 @@ APRS# - Modern APRS software for the amateur radio community
```
Comment (CBielstein) [inline README.md:4]: Add the action's status badge here.

**Pull request merged**